### PR TITLE
ModalMissionComplete Tests

### PR DESCRIPTION
**Pull request by danZzyy**
A draft of the Jasmine tests I wanted to share. The extra commit were commits I was going to amend to but forgot about so please look at my last commit. Several changes were made to supporting modules to make public methods that were previously private so that they can be accessed in testing.

### CODE REVIEW

File: app/views/audit.scala.html [hunk 1]
```diff
@@ -338,7 +338,7 @@ <h1 id="modal-mission-header">Mission</h1>
                             <h3>Mission Labels</h3>
                             <table class="table">
                                 <tr>
-                                    <th class="width-50-percent">Curb Ramp</th>
+                                    <th>Curb Ramp</th>
```
Comment (kotarohara) [inline app/views/audit.scala.html]: Is this something to do with the test?

File: public/javascripts/Admin/build/Admin.js [hunk 2]
```diff
@@ -1,6 +1,211 @@
-function Admin (_, $, c3, turf) {
```
Comment (kotarohara) [inline public/javascripts/Admin/build/Admin.js]: I don't think you made changes to Admin, right? I cannot figure out if these changes are significant. If not, please get rid of this from the PR.

File: public/javascripts/SVLabel/src/SVLabel/mission/Mission.js [hunk 3]
```diff
@@ -355,8 +355,12 @@ function Mission(parameters) {
         }
     }
 
+    function setLabelCounts(labelCounts){
```
Comment (kotarohara) [inline public/javascripts/SVLabel/src/SVLabel/mission/Mission.js]: What is this method? It seems like `labelCoutnsAtCompletion`. If so, please fix.

File: app/views/audit.scala.html (hunk 1 above)
Comment (danZzyy) [inline app/views/audit.scala.html]: No its a class that has no use here so i removed it

File: public/javascripts/Admin/build/Admin.js (hunk 2 above)
Comment (danZzyy) [inline public/javascripts/Admin/build/Admin.js]: These are from the merge from master. Shouldnt appear in new branch I am working on.

**Pull request closed without merging**